lib/active_admin.rb

```diff
@@ -33,6 +33,7 @@ module ActiveAdmin
   autoload :Devise,                   'active_admin/devise'
   autoload :DSL,                      'active_admin/dsl'
   autoload :Event,                    'active_admin/event'
+  autoload :Formatter,                'active_admin/formatter'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.